tests/fixtures/transform_custom.py

```diff
@@ -2,18 +2,19 @@
 from scipy.sparse.csr import csr_matrix
 
 
-def transform(data, transformer):
+def transform(X, y, transformer):
```

Comment:
I don't love having transform behave differently for custom transformers vs everything else....open to ideas about what the right way is to do this (add y=None as an optional arg after transformer? I dunno)

Reply:
....update: we're going with (data, transform, y=None) for now (or maybe `(X, transform, y=None)`?)